src/routes/+layout.svelte

```diff
@@ -2,6 +2,10 @@
   import Header from '../components/Header.svelte';
   import Nav from '../components/Nav.svelte';
   import wave from '$lib/assets/bg-wave.png';
+  import { logInPromise, storeChainInfoPromise } from '$lib/stores';
+
+  $: $logInPromise;
+  $: $storeChainInfoPromise;
```

Comment:
Does this actually work to create a subscription? I'd be afraid it might get optimized away since it doesn't do anything... curious to see the transpiled output Javascript.

Reply:
yep it totally works.